frontend/src/Today.jsx

```diff
@@ -1,9 +1,165 @@
-import React from 'react';
-// import styles from './Today.module.css';
+import React, { useState } from 'react';
+import axios from 'axios';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import {
+  DateTimePicker,
+  MuiPickersUtilsProvider,
+} from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import styles from './Today.module.css';
 
 function Today() {
+  const [open, setOpen] = useState(false);
+  const handleClickOpen = () => { setOpen(true); };
+  const handleClose = () => { setOpen(false); };
+
+  const [dueDate, setDueDate] = useState(null);
+  const [targetDate, setTargetDate] = useState(null);
+  const [remindDate, setRemindDate] = useState(null);
+  const handleDueDateChange = (date) => { setDueDate(date); };
+  const handleTargetDateChange = (date) => { setTargetDate(date); };
+  const handleRemindDateChange = (date) => { setRemindDate(date); };
+
+  const useStyles = makeStyles({
+    root: {
+      '& .MuiPaper-root': {
+        backgroundColor: '#FBF8EC',
+        borderRadius: 10,
+        borderColor: '#BD8B13',
+      },
+    },
+    otherButtons: {
+      color: '#154734',
+      fontSize: 15,
+    },
+    title: {
+      display: 'flex',
+      justifyContent: 'center',
+      color: 'white',
+      fontSize: 60,
+    },
+    adda: {
+      color: '#BD8B13',
+    },
+    task: {
+      color: '#154734',
+    },
+    dates: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    button: {
+      fontSize: 60,
+      color: 'white',
+      backgroundColor: '#BD8B13',
+      height: 70,
+      width: 70,
+      borderRadius: '50%',
+      opacity: 0.6,
+      '&:hover': {
+        backgroundColor: '#BD8B13',
+        opacity: 1,
+      },
+    },
+  });
+
+  async function makePostCall(task) {
+    try {
+      const response = await axios.post('http://localhost:5000/Today', task);
+      if (response.status === 201) {
+        return true;
```

Comment:
Needs to catch 401 for unauthorized access aka user logged out, should theoretically route back to login